search/serverless.yml

```diff
@@ -35,7 +35,7 @@ functions:
       CMR_LB_URL: 'http://localhost'
       CMR_STAC_RELATIVE_ROOT_URL: /stac
       CMR_STAC_PROTOCOL: ${param:protocol, 'http'}
-      GRAPHQL_URL: ${param:graphql-url, 'http://localhost/graphql'}
+      GRAPHQL_URL: ${param:graphql-url, 'http://localhost:3000/dev/api'}
```

Comment:
Is this meant to be 3003? That is the http port in the serverless.yaml file on graphql for local serverless?
`  serverless-offline:
    httpPort: 3003
    lambdaPort: 3004`